SemanticAutocomplete.md

```diff
@@ -46,7 +46,7 @@ The response data is broken up into two parts: the original `query` and the list
 
 | Field | Type | Description |
 |-------|------|-------------|
-| type | String | The type or cannonical form of the suggestion. Returned values include: *keyword, job_title, job_level, occupation, skill, school, company, location, freetext*
+| type | String | The type or cannonical form of the suggestion. Possible returned values include: *keyword, job_title, job_level, occupation, skill, school, company, location, freetext*
```

Comment:
I still think we should say "values are" instead of "values include". Include implies there are other values that we have not listed here.